src/microbe_stage/PatchManager.cs

```diff
@@ -195,20 +185,16 @@ private void HandleCellSpawns(Patch patch)
                 continue;
             }
 
-            var density = Mathf.Max(Mathf.Log(population / 25.0f) * 0.01f, 0.0f);
+            var density = Mathf.Max(
+                Mathf.Log(population / Constants.MICROBE_SPAWN_DENSITY_POPULATION_MULTIPLIER) *
+                Constants.MICROBE_SPAWN_DENSITY_SCALE_FACTOR, 0.0f);
```

Comment:
can this be negative?

Reply:
I don't know, I just left the Mathf.Max call here anyway. I just converted the magic numbers into proper named constants here.